docs/cn/basic/Command-Line-Interface.md

```diff
@@ -1,4 +1,4 @@
-﻿---
+---
```

Comment:
please undo this whitespace change

Reply:
not sure what the whitespace it reverts. But I copied the original text and it still shows. I think the new version should be good